pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -42,8 +43,10 @@ def test_ops_error_str(self):
             with pytest.raises(TypeError):
                 left + right
 
-            with pytest.raises(TypeError):
-                left > right
+            # GH 20829: python 2 comparison naturally does not raise TypeError
```

Comment:
use compat.PY3

Reply:
Done, but as your comment was in the code comment (which didn't change), it remained here after my changes.